src/components/Assemblies/AssemblyTooltip.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+	import type { TooltipProp } from './shared';
+
+	export let tooltipProp: TooltipProp | null = null;
+</script>
+
+{#if tooltipProp}
+	<div
+		style:left="{tooltipProp?.x ?? 0}px"
+		style:top="{tooltipProp?.y ?? 0}px"
```

Comment:
@Th1nkK1D I think you mean this one, I've changed as you suggested after moved to separate component.

Reply:
Waittt, my bad, I write the comment on the PR but forget to submit 🫠🙏 the feedback should appear now.

Reply:
Whoops 😆